tests/integration/models/os.spec.ts

```diff
@@ -649,6 +649,7 @@ describe('OS model', function () {
 					gateway: '5.6.7.8',
 					netmask: '9.10.11.12',
 					version: '2.0.8+rev1.prod',
+					provisioningKeyName: 'provision-key',
```

Comment:
```suggestion
```
If all test cases provide a name, then the is no reason to have a separate one that tests the name was indeed assigned.